website/components/features-list/index.jsx

```diff
@@ -1,7 +1,11 @@
-export default function FeaturesList({ title, items }) {
+export default function FeaturesList({ title, items, intro }) {
   return (
     <div className="g-features-list g-grid-container">
       <h2 className="g-type-display-2">{title}</h2>
+      <div
+        className="intro-container"
+        dangerouslySetInnerHTML={{ __html: intro }}
+      />
```

Comment:
This is a bit of a shoehorn here to support the k8s callout text under the "Why HashiCorp" heading. 

<img width="1392" alt="Screen Shot 2020-12-08 at 11 53 57 AM" src="https://user-images.githubusercontent.com/845983/101515184-148cc680-394c-11eb-8c4c-fb22cdeaf541.png">